bin/docker-image-tool.sh

```diff
@@ -208,8 +208,16 @@ Options:
   -f file               Dockerfile to build for JVM based Jobs. By default builds the Dockerfile shipped with Spark.
   -p file               (Optional) Dockerfile to build for PySpark Jobs. Builds Python dependencies and ships with Spark.
                         Skips building PySpark docker image if not specified.
+                        A sample PySpark Dockerfile is at:
+                        kubernetes/dockerfiles/spark/bindings/python/Dockerfile
+			File must be relative the build path, in dev this is copied from:
+			resource-managers/kubernetes/docker/src/main/dockerfiles
   -R file               (Optional) Dockerfile to build for SparkR Jobs. Builds R dependencies and ships with Spark.
                         Skips building SparkR docker image if not specified.
+			 A sample SparkR Dockerfile is at:
```

Comment:
` A` extra space

Reply:
it looks like the preview shows something different. perhaps space vs tab is shown differently.